zkp/js/integration-test/anon.js

```diff
@@ -98,12 +98,34 @@ describe("main circuit tests for Zeto fungible tokens with anonymity without enc
     );
     console.log("Proving time: ", (Date.now() - startTime) / 1000, "s");
 
-    const success = await groth16.verify(verificationKey, publicSignals, proof);
+    let verifyResult = await groth16.verify(
+      verificationKey,
+      publicSignals,
+      proof,
+    );
+    expect(verifyResult).to.be.true;
     // console.log('inputCommitments', inputCommitments);
     // console.log('outputCommitments', outputCommitments);
     // console.log('senderPublicKey', sender.pubKey);
     // console.log('receiverPublicKey', receiver.pubKey);
-    // console.log('publicSignals', publicSignals);
-    expect(success, true);
```

Comment:
note: the syntax of this check was incorrect, so it didn't enforce the boolean check.